recipe/run_test.sh

```diff
@@ -15,6 +16,7 @@ ncks -O --rgr skl=skl_t42.nc \
         $fin \
         foo.nc
 
+echo "ncks 2"
 ncks -O --rgr grid=grd_2x2.nc \
```

Comment:
The hanging seems to be here, which obviously is a line that wasn't added when I put the commented-out tests back in.  It could be that I'm not being patient enough but this command takes no time at all on my laptop so I wonder if something is wrong.